build/index.d.ts

```diff
@@ -0,0 +1,17 @@
+declare const ReconnectingWebsocket: (url: string, protocols?: string | string[], options?: {
```

Comment:
Is this file autogenerated? if not, don't place it inside the build folder. Move it to ./type-definitions